tutorials/epochs/plot_10_epochs_overview.py

```diff
@@ -1,103 +1,390 @@
+epochs.set_channel_types({'EEG 060': 'eeg'})
+
+###############################################################################
+# Selection in the time domain
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# To change the temporal extent of the :class:`~mne.Epochs`, you can use the
+# :meth:`~mne.Epochs.crop` method. However, if you wanted to *expand* the
+# time domain of an :class:`~mne.Epochs` object, you would need to go back to
+# the :class:`~mne.io.Raw` data and recreate the :class:`~mne.Epochs` with
+# different values for ``tmin`` and/or ``tmax``.
+
+shorter_epochs = epochs.copy().crop(tmin=-0.1, tmax=0.1, include_tmax=True)
+
+for name, obj in dict(Original=epochs, Cropped=shorter_epochs).items():
+    print('{} epochs has {} time samples'
+          .format(name, obj.get_data().shape[-1]))
+
+del shorter_epochs
+
+###############################################################################
+# It is also possible to change the "zero point" that defines the time values
+# in an :class:`~mne.Epochs` object, with the :meth:`~mne.Epochs.shift_time`
+# method. :meth:`~mne.Epochs.shift_time` allows shifting times relative to the
+# current values, or specifying a fixed time to set as the new time value of
+# the first sample (deriving the new time values of subsequent samples based on
+# the :class:`~mne.Epochs` object's sampling frequency).
+
+# shift times so that first sample of each epoch is at time zero
+later_epochs = epochs.copy().shift_time(tshift=0., relative=False)
+print(later_epochs.times[:3])
+
+# shift times by a relative amount
+later_epochs.shift_time(tshift=-7, relative=True)
+print(later_epochs.times[:3])
+
+###############################################################################
+# Note that although time shifting respects the sampling frequency (the spacing
+# between samples), it does not enforce the assumption that there is a sample
+# occurring at exactly time=0.
+#
+#
+# Extracting data in other forms
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#
+# The :meth:`~mne.Epochs.get_data` method returns the epoched data as a
+# :class:`NumPy array <numpy.ndarray>`, of shape ``(n_epochs, n_channels,
+# n_times)``; an optional ``picks`` parameter selects a subset of channels by
+# index, name, or type:
+
+eog_data = epochs.get_data(picks='EOG 061')
+meg_data = epochs.get_data(picks=['mag', 'grad'])
+channel_4_6_8 = epochs.get_data(picks=slice(4, 9, 2))
+
+for name, arr in dict(EOG=eog_data, MEG=meg_data, Slice=channel_4_6_8).items():
+    print('{} contains {} channels'.format(name, arr.shape[1]))
+
+###############################################################################
+# You can also export :class:`~mne.Epochs` data to :class:`Pandas DataFrames
+# <pandas.DataFrame>`. The :class:`~pandas.DataFrame` index will be constructed
+# by converting the time of each sample into milliseconds and rounding it to
+# the nearest integer, and combining it with the event types and epoch numbers
```

Comment:
This is a nice brief description of the Dataframe, especially important because of the multi-index structure.